primitives/src/lib.rs

```diff
@@ -65,9 +65,17 @@ pub const DEFAULT_COMMITTEE_SIZE: u32 = 4;
 pub const DEFAULT_BAN_MINIMAL_EXPECTED_PERFORMANCE: Perbill = Perbill::from_percent(0);
 pub const DEFAULT_BAN_SESSION_COUNT_THRESHOLD: SessionCount = 3;
 pub const DEFAULT_BAN_REASON_LENGTH: u32 = 300;
+
 pub const DEFAULT_CLEAN_SESSION_COUNTER_DELAY: SessionCount = 960;
 pub const DEFAULT_BAN_PERIOD: EraIndex = 10;
 
+/// Version returned when no version has been set.
+pub const DEFAULT_FINALITY_VERSION: Version = 0;
+/// Current version of abft.
+pub const CURRENT_FINALITY_VERSION: u16 = 2;
```

Comment:
One nit. Can you define it as `LEGACY_FINALITY_VERSION + 1` ?

That way, we once and for all avoid any possible bugs here.

Reply:
Sure, have made the necessary change.